### ISSUE TITLE
Any updates? Any other solutions you've been using for testing React components?

### ISSUE THREAD

**Issue by williamcotton** [silvenon/react-demo#1]
Hey, I've also experienced similar difficulties with trying to use `refs` in React components that I'm trying to test with jsdom.

Have you made further progress?

**Comment by silvenon** [comment]
No, unfortunately :cry: Currently in my projects I'm using a single document for all the tests (which doesn't require clearing the cache), which is obviously not ideal, but doesn't cause errors.

If you find a solution, please let me know.

**Comment by silvenon** [comment]
One other solution that you can try is using Karma, I heard that works ok.
